addons/udes_advanced_packaging/tests/test_package_reservation.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+from odoo.exceptions import ValidationError
+from odoo.addons.udes_stock.tests.common import BaseUDES
+
+
+class TestPackageReservation(BaseUDES):
+    def test_reserve_full_package_one_product(self):
+        """Test to reserve a full package for one product
+        when initial demand < package quantity
+        """
+
+        # enable full package reservation
+        self.picking_type_pick.u_reserve_as_packages = True
```

Comment:
We're repeating this same line quite a bit, we should create a `setUpClass` method so we only need to do this once. :)